src/main/java/org/reaktivity/nukleus/socks/internal/stream/client/AcceptStreamProcessor.java

```diff
@@ -0,0 +1,444 @@
+import org.reaktivity.nukleus.socks.internal.metadata.State;
+import org.reaktivity.nukleus.socks.internal.stream.AbstractStreamProcessor;
+import org.reaktivity.nukleus.socks.internal.stream.AcceptTransitionListener;
+import org.reaktivity.nukleus.socks.internal.stream.Context;
+import org.reaktivity.nukleus.socks.internal.stream.Correlation;
+import org.reaktivity.nukleus.socks.internal.stream.types.SocksCommandRequestFW;
+import org.reaktivity.nukleus.socks.internal.stream.types.SocksNegotiationRequestFW;
+import org.reaktivity.nukleus.socks.internal.types.OctetsFW;
+import org.reaktivity.nukleus.socks.internal.types.control.RouteFW;
+import org.reaktivity.nukleus.socks.internal.types.control.SocksRouteExFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.AbortFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.BeginFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.DataFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.EndFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.ResetFW;
+import org.reaktivity.nukleus.socks.internal.types.stream.WindowFW;
+
+public final class AcceptStreamProcessor extends AbstractStreamProcessor implements AcceptTransitionListener
+{
+
+    // Current handler of incoming BEGIN, DATA, END, ABORT frames on the ACCEPT stream
+    private MessageConsumer acceptHandlerState;
+
+    private final Correlation correlation;
+
+    private int sourceWindowBytesAdjustment;
+    private int sourceWindowFramesAdjustment;
+
+    private int connectWindowBytes = 0;
+    private int connectWindowFrames = 0;
+
+    DataFW dataNegotiationRequestFW = null;
+    DataFW dataConnectRequestFW = null;
+
+    boolean isConnectReplyStateReady = false;
+
+    private int slotIndex = NO_SLOT;
+
+    // One instance per Stream
+    AcceptStreamProcessor(
+        Context context,
+        MessageConsumer acceptThrottle,
+        long acceptStreamId,
+        long acceptSourceRef,
+        String acceptSourceName,
+        long acceptCorrelationId)
+    {
+        super(context);
+        // init state machine
+        acceptHandlerState = this::beforeBegin;
+
+        final RouteFW tmpConnectRoute = resolveTarget(acceptSourceRef, acceptSourceName);
+        final String tmpConnectTargetName = tmpConnectRoute.target().asString();
+        correlation = new Correlation();
+        correlation.acceptThrottle(acceptThrottle);
+        correlation.acceptStreamId(acceptStreamId);
+        correlation.acceptSourceRef(acceptSourceRef);
+        correlation.acceptSourceName(acceptSourceName);
+        correlation.acceptCorrelationId(acceptCorrelationId);
+        correlation.acceptReplyStreamId(context.supplyStreamId.getAsLong());
+        correlation.acceptReplyEndpoint(context.router.supplyTarget(acceptSourceName));
+        correlation.acceptTransitionListener(this);
+        correlation.connectRoute(tmpConnectRoute);
+        correlation.connectTargetName(tmpConnectRoute.target()
```

Comment:
It's easier to read if you pull these out, example
```
final string targetName=tmpConnectRoute.target().name()
correlation.acceptReplyEndpoint()
```
Also why cross/introduce new terminology on target vs Endpoint?